### ISSUE TITLE
Owners are permanent. (medium severity)

### ISSUE THREAD

**Issue by Dexaran** [EthereumCommonwealth/ethereum-classic-multisig#7]
### Description

A mechanism to update a list of owners or claim ownership is not presented. It can result in a permanent loss of funds in some specific circumstances. 

For example if all the owners will lose control of their multisig-assigned accounts or disappear (die, for example), then all the funds are lost.
Moreover if `dayLimit` is set to zero, none of the owners will be able to spend funds without a sufficient number of confirmations. In this case, a disappearance of (owners count - `required` + 1)  is sufficient to break the whole functionality of the multisig.

I suppose that it is acceptable in general case. As for special cases, I believe that the funds of the official donation multisig are the heritage of the Ethereum CLassic community. As the result, I think that such a loss of funds is unacceptable.

### Recommendation

Implement a mechanism to manage ownership/ claim ownership after a sufficient period of inactivity (say 5 years) so that owners could replace a disappeared community member with a new one if necessary.

Implement a mechanism of ownership claiming, if the multisig completely lost all owners (say, all the owners are inactive for 10 years). As the multisig funds are the heritage of ETC community, I assume that it would be better if a certain community member could claim them in case of total loss of control over this multisig's funds.

**Comment by drd34d** [comment]
I agree, it makes sense to have such a mechanism for certain use cases as you mentioned
